src/renderer/services/eth2/client/lighthouse/beacon.ts

```diff
@@ -0,0 +1,46 @@
+import {IBeaconApi} from "@chainsafe/lodestar-validator/lib/api/interface/beacon";
+import {bytes32, Fork, number64, SyncingStatus, uint64} from "@chainsafe/eth2.0-types";
+import {HttpClient} from "../../../api";
+import {IBeaconConfig} from "@chainsafe/eth2.0-config";
+import {IBeaconClientOptions} from "../interface";
+import {fromJson} from "@chainsafe/eth2.0-utils";
+
+export enum LighthouseBeaconRoutes {
+    VERSION = "/node/version",
```

Comment:
Maybe the ones that are same in both APIs should be in the same place? Will be easier to keep up and update if it gets standardized.

Reply:
routes or implementation?

Reply:
routes

Reply:
I would keep them separated until api get's standardized because it will be hell to manage changes in api